### PR TITLE
Add sorting entries in the inventory file (#1076)

### PR DESCRIPTION
**Pull request by sbbroot**
* Add sorting hostnames for `Azure` and `Any` provider in the inventory
  file

### CODE REVIEW

File: tests/unit/engine/providers/data/APIProxy_data.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                                 'service-2',
+                                 'service-3',
+                                 'service-4']
+                }
+            }
+        },
+        'version': '1.3.0dev'
+    })
+
+RUNNING_INSTANCES: List[List[Dict]] = [
```
Comment (seriva) [inline tests/unit/engine/providers/data/APIProxy_data.py]: This would be specific to Azure, maybe name it something like RUNNING_INSTANCES_AZURE?

**Review by seriva — APPROVED** [review]
Looks good. Don't care for the TestCase wrapping but that's partially preferences ;)

**Review by seriva — CHANGES_REQUESTED** [review]
Missed the use of unittest in from your imports as PawoG noted. We choose pytest over unittest so pls rewrite these tests in pytests as I dont think we should be using 2 testing frameworks mixed.

**Review by erzetpe — DISMISSED** [review]

**Review by erzetpe — DISMISSED** [review]

**Review by seriva — DISMISSED** [review]

**Review by przemyslavic — DISMISSED** [review]

**Review by przemyslavic — APPROVED** [review]

**Review by erzetpe — APPROVED** [review]

**Pull request merged**